documentation/sphinx/source/command-line-interface.rst

```diff
@@ -439,7 +439,7 @@ The ``tenant`` command is used to view and manage the tenants in a cluster. The
 create
 ^^^^^^
 
-``tenant create <NAME> [tenant_group=<TENANT_GROUP>]``
+``tenant create <NAME> [tenant_group=<TENANT_GROUP>] [assigned_cluster=<CLUSTER_NAME>]``
```

Comment:
This argument should also be documented in the list below, though for now I wasn't sure if we should include it with none of the other metacluster aspects documented. It's probably fine to leave it here, though, given the fdbcli reveals this parameter through its help.